lib/internal/Magento/Framework/View/Page/Config.php

```diff
@@ -539,6 +544,31 @@ public function addBodyClass($className)
         return $this;
     }
 
+    /**
+     * Retrieve the additional meta assets
+     *
+     * @return array
+     */
+    public function getAdditionalMetaAssets()
+    {
+        return $this->additionalMetaAssets;
+    }
+
+    /**
+     * Adjust metadata content url
+     *
+     * @param string $content
+     * @return string $content
+     */
+    public function getMetaAssetUrl($content)
+    {
+        $parsed = parse_url($content);
```

Comment:
Hi can you add the second parameter, please

Reply:
Hi @larsroettig, I've seen you marked as resolved the previous comment and thought its not needed. 
Will add it. Thank you!